python/paddle/vision/ops.py

```diff
@@ -835,6 +836,123 @@ def forward(self, x, offset, mask=None):
         return out
 
 
+def distribute_fpn_proposals(fpn_rois,
```

Comment:
shall we delete function of `paddle.fluid.layers.distribute_fpn_proposals` and modify `paddle.fluid.layers.distribute_fpn_proposals` import here, can discuss with @zhiboniu

Reply:
Done.
Through the discusstion with @zhiboniu , current now we change implement of the corresponding fluid api to using `paddle.vision.ops.distribute_fpn_proposals`. And we still keep this fluid api to avoid potential incompatibilities.